rosdep/python.yaml

```diff
@@ -392,6 +392,12 @@ python-certifi:
     xenial: [python-certifi]
     yakkety: [python-certifi]
 python-chainer:
```

Comment:
Since this is pip only, I'd suggest we update it to use the `-pip` extension as we generalize it.